_sass/variables.scss

```diff
@@ -4,10 +4,13 @@ $accent-color: #d500f9;
 
 @import url('https://fonts.googleapis.com/css?family=Signika:300,400,500,700|Raleway:300,400,600');
 
-// @font-face {
-//   font-family: "Anderson_Grotesk";
-//   src: url("{{ site.url }}assets/fonts/AndersonGrotesk.otf") format("opentype");
-// }
+@font-face {
+   font-family: "Anderson_Grotesk";
+   src: url("fonts/AndersonGrotesk.otf") format("opentype");
+}
 
-$title-font: 'Signika', sans-serif;
-$content-font: 'Raleway', sans-serif;
+//$title-font: 'Signika', sans-serif;
+//$content-font: 'Raleway', sans-serif;
+
+$title-font: 'Anderson_Grotesk', sans-serif;
```

Comment:
If we're only going to have one font, could you collapse this to just `$font` and use that as the font variable?

Reply:
I can do this, but does this mean I have to do a find & replace in the other 4 .scss files, or is there a better way?

Reply:
I think that's what we'd have to do 😞 In sublime that should just be a `command + shift + f` and then replace

Reply:
thumbs up